src/components/reusable/blockCodeView/blockCodeView.ts

```diff
@@ -47,15 +46,11 @@ const LANGUAGE_SPECIFIC_TOKENS: Record<string, string[]> = {
  */
 @customElement('kyn-block-code-view')
 export class BlockCodeView extends LitElement {
-  static override styles = [
-    BlockCodeViewStyles,
-    ShidokaLightTheme,
-    ShidokaDarkTheme,
-  ];
+  static override styles = [BlockCodeViewStyles, ShidokaSyntaxTheme];
 
   /** Sets background and text theming. */
   @property({ type: String })
-  darkTheme: 'light' | 'dark' = 'dark';
+  darkTheme: 'light' | 'dark' | 'default' = 'default';
```

Comment:
I notice when changing this, it's changing the theme of the entire page. Looks like it's targeting the meta tag? We don't want that to happen here. This should really be an override which only applies to the code snippet itself. That can be achieved by using CSS `color-scheme: <value>` on the container element.

Reply:
i refactored both inline and block components and how they're handling this separation and prioritization of theming coming from the global `color-scheme` vs. the component level `darkTheme` property. this should be fixed on the deploy preview now.

Reply:
It seems like it's scoped correctly now, however the "default" theme is appearing to me as light even though I have auto set overall and my system preference is dark.

Reply:
i'm not sure all of this prefers logic is necessary, maybe there's already an easier way to achieve this built into the codebase, but let me know your thoughts on:

https://github.com/kyndryl-design-system/shidoka-applications/blob/9529158a46665ad4a5b3707606059088f12d8a58/src/components/reusable/blockCodeView/blockCodeView.ts#L158-L188

Reply:
I don't think so. By default, you shouldn't have to do anything and it will use the application-level theme. All you should need to do to override that is set CSS `color-scheme: dark/light` on the container.